custom_metrics/privacy.js

```diff
@@ -14,4 +15,10 @@ return JSON.stringify({
       a.innerText.match(pattern)
```

Comment:
By the way, do you care to know which privacy text matched, or is it sufficient to know that any one of them matched?

Reply:
We are sufficient with just finding a proper threshold on the number of matched links, probably looking at >0.
Obviously there may be some legal topic websites that have plenty of links, and some localised titles that were missed here.

I was actually thinking to just add a.innerText to wpt_bodies.anchors and run matching in BQ, but trying to keep data small.
Do you think it makes more sense because links information can be reused for other cases?

Reply:
I wouldn't recommend dumping all anchor text, but if you want to return the matches from this narrow list of interest that makes sense to me so you can graph the popularity of different privacy phrases.

I'll merge this PR as-is so we can get something in before the cutoff, but please submit a PR ASAP if you have any additional enhancements. I'll be syncing the test server with the custom metrics tomorrow.